src/controllers/crossCutting.controller.js

```diff
@@ -6,18 +6,18 @@ const exportFinalVideo = require("../services/exportFinalVideo");
 const getFinalVideoUrl = require("../services/getFinalVideoUrl");
 const progressStatus = require("../routes/progressStatus");
 const { TEMP_DIR_FRAMES, TEMP_DIR } = require("../constants/paths");
+const assignEditPoints = require("../services/assignEditPoints");
 
 async function crossCutting(req, res, next) {
   progressStatus.stage = "editing";
-  const { isVertical } = req.body;
   const { selectedEditPoints } = req.body;
   const { singleShots } = res.locals;
   const { editPoints } = res.locals;
 
-  const { subOneMergedFrames, subTwoMergedFrames } = await sortFrames(
-    singleShots,
-    editPoints,
-  );
+  const { subOneMergedFrames, subTwoMergedFrames } =
+    selectedEditPoints === undefined
+      ? await sortFrames(singleShots, editPoints)
+      : assignEditPoints(editPoints, selectedEditPoints);
```

Comment:
편집점이 없으면 sortFrames를 있다면 assignEditPoints로직을 통해서 app 로직과 웹 로직을 분리해주셨군요.
selectedEditPoints가 정의되었는지, 정의되지 않았는지 확인하고 있기때문에 꼭 undefined를 쓰지 않고도 처리는 가능할 것 같습니다!
```suggestion
  const { subOneMergedFrames, subTwoMergedFrames } =
    selectedEditPoints
    ? assignEditPoints(editPoints, selectedEditPoints)
    : await sortFrames(singleShots, editPoints);
```

Reply:
`selectedEditPoints` 즉, 선택된 편집점이 없을 경우 값이 반드시 `undefined`로 온다면 경우라면 확실하게 명시하여 확인하는 것도 좋을 것 같다는 의견입니다! 다만 현재 app 클라이언트 코드를 확인해보니 `editPoints`가 선택되지 않을 경우 `undefined`를 반환하는지 여부에 대해서는 한번 더 확인 부탁드립니다! (빈 배열로 오는 것 같은데 확인 부탁드립니다!🙇‍♀️)

(만약 빈 배열로 오고 이를 처리해야한다면 제가 추후 진행할 리팩토링시 진행해보겠습니다!)

Reply:
`selectedEditPoints`가 없다면 키는 있으나 값이 빈 배열입니다. 빈 배열일 경우라도 로직이 정상작동해서 이대로 유지하겠습니다!